src/stories/StyledButton.tsx

```diff
@@ -1,4 +1,5 @@
 import StyledLoadingSpinner, { SpinnerSizes, SpinnerVariant } from './StyledLoadingSpinner';
+import DfxIcon, { IconColors, IconSizes, IconVariant } from './DfxIcon';
 
 export enum StyledButtonSizes {
```

Comment:
Not related to this PR, but enum names should be singular (e.g. `StyledButtonSize`). Same for `StyledButtonColor`.